examples/pebbleui/index.html

```diff
@@ -0,0 +1,141 @@
+    <!-- /build -->
+    <!-- build:css ../../css/style.css -->
+    <link href="../../html/css/style.css" rel="stylesheet">
+    <!-- /build -->
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <!-- Include PebbleUI Rocky Module -->
+    <script src="js/modules/PebbleUI.js"></script>
+    <link rel="stylesheet" href="../common/css/style.css">
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+        <h1>Pebble UI</h1>
+        <p class="lead">
+            This example shows how a UI layer can be built on top of Rocky, and used to build more complex applications.
+        </p>
+    </div>
+
+    <div class="col-md-6">
+        <canvas id="pebble" class="rocky" width="432" height="504"></canvas>
+    </div>
+
+    <div class="col-md-6">
+        <h2>What's going on?</h2>
+        <p>
+            This example demonstrates how to use PebbleUI to create a simple application with cards, menus, and general windows.
+        </p>
+        <p>View <a href="js/modules/PebbleUI.js">PebbleUI.js</a> for more details.
+    </div>
+    <!-- build:template
+    <%= github_banner %>
+    /build -->
+</div>
+</body>
+
+<script>
+(function() {
+  rocky = Rocky.bindCanvas(document.getElementById('pebble'));
+  rocky.export_global_c_symbols();
+
+  var UI = new PebbleUI(rocky);
+
+  var graphics = new UI.Window({
+    backgroundColor: rocky.GColorBlack
+  }).add(new UI.Circle({
+    bounds: [0, 0, 144, 168],
+    backgroundColor: rocky.GColorRed
+  })).add(new UI.Line({
+    color: rocky.GColorOxfordBlue,
+    width: 2,
+    bounds: [10, 10, 60, 50]
+  })).add(new UI.Rect({
+    bounds: [70, 125, 60, 30],
+    backgroundColor: rocky.GColorBlue
+  })).add(new UI.Text({
+    bounds: [30, 30, 104, 108],
+    text: 'This is some text or whatever, blah blah blah.. ' +
+          'press back to go back a screen',
+    color: rocky.GColorYellow
+  })).add(new UI.Image({
+    bounds: [10, 10, 25, 25],
+    url: 'https://www.ciee.org/high-school-summer-abroad/' +
+          'images/sitewide/homeBtn25x25.png',
+    z: -1
+  }));
+
+  var menu = new UI.Menu({
+    title: 'Fruit List',
+    backgroundColor: rocky.GColorBlack,
+    selectedItemColor: rocky.GColorBlack,
+    selectedItemDetailsColor: rocky.GColorBlack,
+
+    titleColor: rocky.GColorWhite,
+    selectedItemBackgroundColor: rocky.GColorWhite,
+    itemColor: rocky.GColorWhite,
+    itemDetailsColor: rocky.GColorWhite
+  }).onSelected(function(item) {
+    // Dynamically create a card
+    new UI.Card({
+      title: item.title,
+      subtitle: item.details
+    }).show();
+  });
+
+  menu.push({
```

Comment:
do you also want to have feedback on the API design? I think that the intend of `#push` on a menu directly is not clear.